ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ParentBeaconBlockRootHelper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.mainnet;
+
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.account.MutableAccount;
+import org.hyperledger.besu.evm.worldstate.WorldUpdater;
+
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.units.bigints.UInt256;
+
+/** A helper class to store the parent beacon block root. */
+public class ParentBeaconBlockRootHelper {
+
+  // Modulus use to for the timestamp to store the  root
+  public static final long HISTORICAL_ROOTS_MODULUS = 98304;
+
+  // Address of the system user, that is used to call the contract for storing the root
+  // public static final Address SYSTEM_ADDRESS =
+  // Address.fromHexString("0xfffffffffffffffffffffffffffffffffffffffe");
+
+  // The address of the contract that stores the roots
+  // public static final Address BEACON_ROOTS_ADDRESS =
+  // Address.fromHexString("0x89e64Be8700cC37EB34f9209c96466DEEDc0d8a6");
```

Comment:
are these comments staying - if so does it make sense to javadoc them so they're more visible

Reply:
This address is the one that they will probably use in mainnet. We have to set it once they decide how to deploy the contract.